### PR TITLE
Fix word length

### PR DESCRIPTION
**Pull request by GuilhermeTavares13**
Iniatiated a new PR. Fixed changed files.
Proposition to fix the word length.
While using the english dictionary the number of tiles wasn't the same as the number of letters.

### CODE REVIEW

File: src/dictionary.js [hunk 1]
```diff
@@ -10,14 +10,15 @@ class Dictionary {
     constructor(words) {
         this.length = undefined;
         this.words = [];
+        let word = '';
```
Comment (clupasq) [inline src/dictionary.js:13]:
```suggestion
```

File: src/dictionary.js [hunk 2]
```diff
@@ -10,14 +10,15 @@ class Dictionary {
...
         for (const w of words) {
+            word = w.replace(/(?:\\[rn]|[\r\n]+)+/g, "");
```
Comment (clupasq) [inline src/dictionary.js:15]:
```suggestion
            const word = w.replace(/(?:\\[rn]|[\r\n]+)+/g, "");
```

**Review by clupasq — CHANGES_REQUESTED** [review]
I propose a slightly improved version, in which the `word` variable is restricted to a narrower context.


Also, what words from the English dictionary did not have the correct length?
I just checked the en dictionary and have a hard time understanding why this change is needed.

**Comment by GuilhermeTavares13** [comment]
Thank you for the insight.
While using it I noticed that the number of letters was wrong, because the carriage return and new line was considered. There is an old PR mentioning this problem, but I think it was closed and dismissed. I will consider your proposition more carefully and improve my code. 
In case you couldn't simulate this problem, I will stop my PR process and start to assume that is only my problem.

**Pull request closed without merging**